native/templates/openharmony/entry/package-lock.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "entry",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "@types/libentry.so": {
+      "version": "file:src/main/cpp/types/libentry",
+      "dev": true
+    }
+  }
```

```suggestion
  "dependencies": {}
```

we can remove this dependency